cmd/tink-worker/tink/tink.go

```diff
@@ -0,0 +1,61 @@
+package tink
```

Comment:
I think the target for this code should be adding it into `cmd/tink-worker/client/tink.go`, and moving the tests into `cmd/tink-worker/client/tink_tests.go`.

Reply:
but As per this https://github.com/jacobweinstock/proposals/tree/0027-tink-worker-rearch/proposals/0027  this is in tink package ?

Reply:
I won't block the PR for this, but I think we're either going to need to move all of the code into one or the other of these locations. We can do that at a later time though.